stdlib/tkinter/__init__.pyi

```diff
@@ -319,6 +319,9 @@ class _GridIndexInfo(TypedDict, total=False):
     uniform: str | None
     weight: int
 
+class _BusyInfo(TypedDict):
+    cursor: _Cursor
```

Comment:
Doing this to appease stubtest.

Reply:
I think I'd prefer to spell out the kwargs explicitly and ignore stubtest; it probably gives a better experience for users calling this function.